docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,110 @@
+In this example workorders are accident reports to be filled out at the seen of an accident.
+
+image::/shared/images/Workorder-mobile.png[Workorders]
+
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::/shared/images/sidebar.png[Workorders]
+
+Return to the Workorders screen and click on an one of the workorders. The following screen will render
+
+image::/shared/images/workorder-overview.png[overview]
+
+Click on `BEGIN WORKFLOW` to proceed with data entry in the following `Accident Report`.
+
+image::/shared/images/workorder-accidentReport.png[accident report]
+
+Once the `Accident Report` form is completed click on `CONTINUE` on the bottom of the screen to proceed to the 
+`Vehicle Inspection` form. The form asks for checks for fuel ,tires and lights.
+
+image::/shared/images/workorder-vehicleInspection.png[vehicle inspection]
+
+Once the `Vehicle Inspection` form is completed click on `CONTINUE` on the bottom of the screen to proceed to the 
+`Summary` which shows the status of the workorder.
+
+image::/shared/images/workorder-summary.png[summary]
+
+Once on the summary screen you have finished your submission . To proceed click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen. On the Workorders screen you will see that the workorder is now marked with a green tick to so completion.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+The login screen is as follows
+
+image::/shared/images/login-portal.png[login portal]
+
+Onced logged in you will be presented with the following screen for admistration of workorders and workflows.
+
+image::/shared/images/portal.png[portal]
+
+From this screen you can create new workorders by clicking the `NEW WORKORDER` button on the right panal. Once clicked
+it will render the following form for creating a new workorder.
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new workorder will appear in the list on middle panal after a refresh. 
+Once the workorder is created it will sync to the individual worker mobile that it was assigned to. To edit individual workorders
+click the workorder from the center panal list to get the following screen.
+
+image::/shared/images/portal-viewWorkorder.png[view workorder]
+
+In the top right hand side there is a trash icon to allow you to delete a workorder. The pencil icon is beside 
+this allows you to update a workorder's title,assignee and select a workflow, as with creating a workorder these changes will sync to 
```

Comment:
This statement may not be right (as people can modify this now). 
Let's avoid listing fields.